charts/README.md

```diff
@@ -0,0 +1,90 @@
+# Install with Helm 
+
+## Deploy cert-manager chart
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
+
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.12.0 \
+  --set installCRDs=true
+```
+
+##  Install kube-green chart 
+
```

Comment:
Since it could change with the time, to improve the maintainability we could set it as:

```suggestion
##  Install with kube-green chart 

To successfully install kube-green, in the cluster must be installed a cert-manager.
If it is not already installed installed, [check the cert-manager installation guide](https://cert-manager.io/docs/installation/).

To install kube-green using the helm-chart (inside the `kube-green` namespace), clone the kube-green repository and run this command:

```